_includes/footer.html

```diff
@@ -35,9 +35,9 @@ <h4>About the School</h4>
     </div>
     <div>
         <h4><a href="/our-programs">Our Programs</a></h4>
-        <a href="/semi-intensive-and-intensive-programs">Semi-Intensive/Intensive Programs</a>
+        <a href="/intensive-programs">Semi-Intensive/Intensive Programs</a>
         <a href="/conversation-courses">Conversation Courses</a>
```

Comment:
/conversation?

Reply:
I think /intensive and /conversation are good, but would do better under a /programs section. So something like /programs/intensive.

Reply:
Hm. That is one option. I'm okay with either that (as long as it's consistent among the programs) or just having plain /intensive and /conversation. Also, /our-programs should become the root, i.e. /programs.